src/io-methods/azure.js

```diff
@@ -27,8 +29,8 @@ async function createBlobService() {
   }
 }
 
-async function getBlobService() {
-  blobService = blobService || (await createBlobService());
+async function getBlobService(target) {
+  blobService = blobService || (await createBlobService(target));
```

Comment:
This looks to have a bug. The `blobService` variable will be set up for a single Azure storage blob and then be reused for all subsequent requests, even if they are for a different location in the config file.

What this means is that if you call the deployer first for stage and then for prod that the call to prod will update the stage import map.

To fix, it might be best just to recreate the blob every time:

```suggestion
  blobService = await createBlobService(target);
```

With this change ^ it also might make sense to delete the `getBlobService` function entirely, since it's just a pass-through to `createBlobService`